### PR TITLE
Fortran interfaces

### PR DESCRIPTION
**Pull request by victorapm**
This PR adds fortran interfaces for `hypre_MGR` and `hypre_ILU`.

### CODE REVIEW

**Review by rfalgout — APPROVED** [review]
Looks great!  Thanks!

File: src/parcsr_ls/F90_HYPRE_parcsr_mgr.c [hunk 1]
```diff
@@ -0,0 +1,807 @@
...
+void
+hypre_F90_IFACE(hypre_mgrdirectsolversolve, HYPRE_MGRDIRECTSOLVERSOLVE)
+( hypre_F90_Obj *solver,
+  hypre_F90_Obj *A,
+  hypre_F90_Obj *b,
+  hypre_F90_Obj *x,
+  hypre_F90_Int *ierr )
+{
+   *ierr = (hypre_F90_Int)
+           ( HYPRE_MGRDirectSolverSolver(
```
Comment (oseikuffuor1) [inline src/parcsr_ls/F90_HYPRE_parcsr_mgr.c]: Should this be HYPRE_MGRDirectSolverSolve instead?

Comment (victorapm) [reply src/parcsr_ls/F90_HYPRE_parcsr_mgr.c]: Thanks for catching it! :)

**Review by oseikuffuor1 — APPROVED** [review]
Looks good Victor. Thanks.

**Pull request merged**